hadoop-hdfs-project/hadoop-hdfs/src/main/java/io/hops/transaction/lock/LeaseLock.java

```diff
@@ -89,4 +99,16 @@ protected void acquire(TransactionLocks locks) throws IOException {
   protected final Type getType() {
     return Type.Lease;
   }
+
+  public void setSingleFileLock(String path){
```

Comment:
Why not to use another constructor with singleFileLock and set it to be final.

Reply:
Sire I will create a new class

Reply:
refactored